src/test/java/org/myrobotlab/service/NeoPixelTest.java

```diff
@@ -5,30 +5,32 @@
 
 import static org.junit.Assert.assertTrue;
 
-import org.junit.Before;
+import org.junit.BeforeClass;
```

Comment:
why BeforeClass instead of just Before?

Reply:
Before is before every test
BeforeClass is before all tests once